modules/imaging_browser/php/imaging_browser.class.inc

```diff
@@ -56,214 +56,50 @@ class Imaging_Browser extends \NDB_Menu_Filter
-                    // Display only the phantom scans at the user's centers
-                    // For phantoms, use session centerID
-                    $this->query .= " AND s.CenterID IN (" . $site_arr . ") ";
-                }
-            } else {
-                if ($user->hasPermission('imaging_browser_phantom_allsites')) {
-                    $this->query .= " AND (c.RegistrationCenterID IN ($site_arr) 
-                                      OR c.Entity_type = 'Scanner') ";
-                } else if ($user->hasPermission('imaging_browser_phantom_ownsite')) {
-                    $this->query .= " AND (c.RegistrationCenterID IN ($site_arr)
-                                      OR (c.Entity_type = 'Scanner'
-                                      AND s.CenterID IN (" . $site_arr . "))) ";
-                } else {
-                    $this->query .= " AND c.RegistrationCenterID IN ($site_arr) ";
-                }
-            }
-        }
-
-        // Initialize the arrays that go into this->columns and
-        // $this->headers so that array_merge works even if no
-        // $scan_types are set in the configuration module
-        $modalities_subquery_as = array();
-        $as = array();
-
-        foreach ($case_desc as $key => $value) {
-            $modalities_subquery[$key] = $value;
-            $as[$key] = $scan_id_types[$key] . '_QC_Status';
-            $modalities_subquery_as[$key] = $modalities_subquery[$key] .
-                ' as ' .
-                $as[$key];
-        }
+        return false;
+    }
 
-        $this->columns = array(
-                          'p.Name as Site',
-                          'c.PSCID as PSCID',
-                          'c.CandID as DCCID',
-                          '(SELECT Name 
-                            FROM Project 
-                            WHERE ProjectID=c.ProjectID) as project',
-                          's.visit_label as Visit_Label',
-                          "$PendingFailSubquery as Visit_QC_Status",
-                          'MIN(md.AcquisitionDate) as First_Acquisition',
-                          'FROM_UNIXTIME(MIN(f.InsertTime)) as First_Insertion',
-                          'FROM_UNIXTIME(MAX(fqc.QCLastChangeTime)) as Last_QC',
-                          "$NewDataSubquery as New_Data",
-                         );
-        $this->columns = array_merge(
-            $this->columns,
-            array("GROUP_CONCAT(DISTINCT OutputType) as Links"),
-            array(
-             's.ID as sessionID',
-             'GROUP_CONCAT(DISTINCT modality.Scan_type) as sequenceType',
-             "$PendingNewquery as pending",
-            ),
-            $modalities_subquery_as
-        );
+    /**
+     * Tells the base class that this page's provisioner can support
+     * the UserSiteMatch filter.
+     *
+     * @return bool always true
+     */
+    public function useImagingBrowserSiteFilter() : bool
```

Comment:
This doesn't seem to be used anywhere

Reply:
sorry... Forgot to commit DataFrameworkMenu file in the last commit. The next commit will have those changes.